docs/DeveloperGuide.md

```diff
@@ -1,18 +1,149 @@
-# Developer Guide
+# Developer Guide - Diet Manager
 
-## Design & Implementation
+## Content
+1. [Design](#design)
+2. [Implementation](#implementation)
```

Comment:
It would be great to include Sub-headings in the content page for readability! anyway, good job on the clickable content page 👍